etcdserver/v3demo_server.go

```diff
@@ -96,6 +96,8 @@ func (s *EtcdServer) Compact(ctx context.Context, r *pb.CompactionRequest) (*pb.
 	result, err := s.processInternalRaftRequest(ctx, pb.InternalRaftRequest{Compaction: r})
 	if r.Physical && result.physc != nil {
 		<-result.physc
+		// if server goes down, should not replay the compaction
+		s.be.ForceCommit()
```

Comment:
i do not quite understand the relationship between the comment and the code. can we be more clear?